vessel-runtime/src/main/java/com/textnow/android/vessel/Vessel.kt

```diff
@@ -44,6 +44,38 @@ interface Vessel {
     @VisibleForTesting
     fun close()
 
+    /**
+     * Read the entire database into the cache in one operation.
+     *
+     * This requires a cache.  If no cache was configured, this function does nothing.
+     *
+     * This can lead to a significant speedup in some scenarios:
+     * - When reading many keys
+     * - When writing many keys, but the values written match what is already in the database
+     *
+     * Many random reads can be slower than one larger read.  Similarly one large read is likely
+     * faster than re-writing many values.
+     */
+    suspend fun preload()
```

Comment:
since the interface is changing, we'll want to mark this as a breaking change in case anyone had their own implementation.  don't forget to update the CHANGELOG.